src/sentry/utils/javascript.py

```diff
@@ -185,7 +185,7 @@ def transform(self, obj, request=None):
             'id': six.text_type(obj.id),
             'count': six.text_type(obj.times_seen),
             'title': escape(obj.title),
-            'message': escape(obj.message_short),
+            'message': escape(obj.get_legacy_message()),
```

Comment:
it was wrong to begin with. I actually plan to follow up and remove this utility as its super old and not kept up to date.